### ISSUE TITLE
ota.py errors when searching for espurna devices

### ISSUE THREAD

**Issue by CrappyTan** [xoseperez/espurna#2287]
Hi,

Getting error when trying to run ota to search for devices to update.

It does not work with python2.7 or 3.7

![image](https://user-images.githubusercontent.com/27884877/85636763-44cff980-b679-11ea-932a-f6c8157d6005.png)

What am I doing wrong?

**Comment by mcspr** [comment]
zeroconf stuff changed, perhaps?
does `pip install zeroconf==0.24.5` fix this? (or 0.24.0, see requirements.txt)

**Comment by mcspr** [comment]
Quoting changelog at https://github.com/jstasiak/python-zeroconf

> 0.27.0
> ...
> Removed previously deprecated ServiceInfo address constructor parameter and property

I also found this patch https://github.com/home-assistant-libs/pychromecast/commit/6c07e0327fd87de3b1e9511c843bc6ef166dc5b8#diff-c86c725f2be6a5290140ac27b5c96835, which looks something like this for ota.py code:
```diff
diff --git a/code/ota.py b/code/ota.py
index 3541fcbb..c7750146 100755
--- a/code/ota.py
+++ b/code/ota.py
@@ -113,9 +113,11 @@ class Listener:
             return

         hostname = info.server.split(".")[0]
+        addresses = info.parsed_addresses()
+
         device = {
             "hostname": hostname.upper(),
-            "ip": socket.inet_ntoa(info.address),
+            "ip": addresses[0] if addresses else info.host,
             "mac": "",
             "app_name": "",
             "app_version": "",
```
